src/Mod/Part/Gui/Utils.h

```diff
@@ -0,0 +1,29 @@
+/***************************************************************************
```

Comment:
This is the old license block, please use: https://freecad.github.io/DevelopersHandbook/codeformatting/#c-copyright-header